python-package/xgboost/core.py

```diff
@@ -1347,7 +1353,7 @@ class Booster:
 
     def __init__(
         self,
-        params: Optional[Dict] = None,
+        params: Optional[Union[Dict, List]] = None,
```

Comment:
I think you have defined an alias in the _typing module.

Reply:
Maybe this can be fixed together with the previous comment, by introducing `BoosterParam`.